packages/developer-console-spa/src/@types/Feedback/types.d.ts

```diff
@@ -9,6 +9,45 @@ declare type FeedbackConfig = {
   feedbackEmail: string;
 };
 
+declare enum FeedbackCategory {
+  BUG = "BUG",
+  FEEDBACK = "FEEDBACK",
+}
+
+declare enum FeedbackStatus {
+  CLOSED = "CLOSED",
+  OPEN = "OPEN",
+}
+
+type FeedbackUserProfile = {
+  cn: String;
+  name: string;
+  uid: string;
+  rhatUUID: string;
+  email: string;
+  url: string;
+};
+
+declare type FeedbackType = {
+  id: string;
+  summary: string;
+  description: string;
+  experience: string;
+  error: string;
+  config: string;
+  state: string;
+  source: string;
+  ticketUrl: string;
+  category: FeedbackCategory;
+  module: string;
+  stackInfo: StackType;
+  assignee: FeedbackUserProfile;
+  createdOn: string;
+  createdBy: string | FeedbackUserProfile;
+  updatedOn: string;
+  updatedBy: string | FeedbackUserProfile;
```

Comment:
Conflict with the timestamp time for createdOn and updatedOn

Reply:
Sorry, didn't get you